src/Directory.Build.props

```diff
@@ -0,0 +1,42 @@
+<Project>
+  <ItemGroup>
+    <SourceRoot Include="$(MSBuildThisFileDirectory)/"/>
+  </ItemGroup>
+  <PropertyGroup>
+    <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+
+    <Company>NLog</Company>
+    <Authors>Jarek Kowalski,Kim Christensen,Julian Verdurmen</Authors>
```

Comment:
This might not be true for all new nuget-packages

Reply:
true, I will remove the directory.props solution